### PR TITLE
feature(developers): Devs can show loaded AMD modules in console

### PR DESCRIPTION
**Pull request by mrclay**

### CODE REVIEW

**Comment by ewinslow** [comment]
Cool

**Comment by mrclay** [comment]
Does cool mean merge?

File: mod/developers/views/default/js/elgg/dev.js [hunk 1]
```diff
@@ -4,15 +4,11 @@
...
-		$("#developers-inspect-results").jstree({
-			"plugins" : [ "themes", "html_data" ],
-			"themes" : {"icons" : false}
-		}).bind("loaded.jstree", function() {
-			$("#developers-inspect-results").fadeIn();				
-		});
-	}
-	
-	setupInspectTree();
+	// setup inspect tree
```
Comment (hypeJunction) [inline mod/developers/views/default/js/elgg/dev.js]: Does this belong in this commit?

File: mod/developers/views/default/js/elgg/dev/amd_monitor.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// http://stackoverflow.com/a/19043564/3779
+var __amd_monitor_modules = require.s.contexts._.defined;
```
Comment (hypeJunction) [inline mod/developers/views/default/js/elgg/dev/amd_monitor.js]: Do we need to keep this in the global scope?

**Comment by hypeJunction** [comment]
LGTM

**Comment by mrclay** [comment]
I addressed the comments

**Comment by hypeJunction** [comment]
I think you added more stuff that's from another commit...

**Pull request merged**